docs/variants/qemu_q35/development.md

```diff
@@ -0,0 +1,121 @@
+# Dasharo (UEFI) v0.1.0 for QEMU Q35
+
+
+1.  Please verify your qemu installation, and then invoke the qemu emulator with the built OVMF firmware image as given below:
+    
+	```bash 
+	qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd -nographic -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
+	```
+	
+	*  The `-drive` parameter indicates that the device is a pflash with the firmware image file pointing out to the built OVMF.fd image.
+	
+	* `-nographic` indicates to start the qemu emulator without any graphical output. This is useful while testing in the environment like docker.
+	
+	* `-debugcon file:debug.log -global isa-debugcon.iobase=0x402` default OVMF build writes debug messages to IO port **0x402**. The following qemu command line options save them in the file called debug.log.
+	
+	* `-global ICH9-LPC.disable_s3=1` SMM is put to use in the S3 suspend and resume infrastructure, and in the UEFI variable driver stack. Similarly, a pflash-backed variable store is a requirement. As SMM was enabled while building the firmware image.
+	
+	* `-net none` disables the OptionROM execution which will interfere with the firmware image, if not disabled.
+	
+	* `-machine q35,smm=on` For SMM to work, only Q35 machines are supported hence the machine type.
+
+2. After executing the above qemu command, one should be able to see the UEFI built-in shell and should be able to get to the BIOS selection area.
+
+3. The features which are enabled in the `OvmfPkgX64.dsc` can be verified in the `BIOS menu` and also at the `Device Manager section` and the Dasharo features can be verified at the `Dasharo System Features` section.
+
+
+
+## Useful Tips for modifying the changes in .DSC & .FDF
+
+
+### Following the below guide, one can rebuild the firmware image and run it in the qemu and experiment with the features which are needed.
+
+
+1.  By making some changes in the `OvmfPkgX64.dsc` file, one can add features like OPAL, SATA security support for QEMU Q35 machine and also for specific platforms. 
```

Comment:
Maybe these options should be permanently enabled... I mean we will always want to have OVMF fully 
featured, mainly for testing.

What I really meant by tips is how to quickly rebuild if you make changes in C files mostly.

Reply:
I agree, that the Dasharo Features should be enabled. But for someone who is building the Dasharo Ovmf, I thought it would be worth noting them to point where they can enable or disable the features.